templates/create-frontend-plugin/docs/features/api.md

```diff
@@ -1,3 +1,5 @@
+# Guide to Consuming APIs in your UI Components
```

Comment:
When this file is rendered by techdocs , this title appears twice
Which is why I removed this in  the first place

Reply:
For me it looks like they show the title from the navigation only when there is no h1 in the first line.. Hmm :man_shrugging: